src/Files.App/Views/ShellPanesPage.xaml.cs

```diff
@@ -343,14 +344,35 @@ private void RemovePane(int index = -1)
 			// Get proper position of sizer that resides with the pane that is wanted to be removed
 			var childIndex = index * 2 - 1;
 			childIndex = childIndex >= 0 ? childIndex : 0;
+			if (childIndex + 1 > RootGrid.Children.Count)
+				return;
+
+			if (childIndex is 0)
+			{
```

Comment:
```suggestion
			if (childIndex >= RootGrid.Children.Count)
				return;

			if (childIndex == 0)
			{
```
It may be a matter of taste, but this is easier for me to understand.